test/TradeTrustERC721.js

```diff
@@ -65,4 +69,84 @@ contract("TradeTrustErc721", accounts => {
       /VM Exception while processing transaction: revert ERC721: transfer caller is not owner nor approved/
     );
   });
+
+  describe("Surrendered TradeTrustERC721 Work Flow", () => {
+    let tokenRegistryInstanceWithShippingLineWallet;
+    let tokenRegistryAddress;
+    let escrowInstance;
+    let escrowInstanceAddress;
+    let escrowInstance1;
+    let escrowInstanceAddress1;
+
+    beforeEach("", async () => {
+      // Setup in accordance to workflow, starting test after the point of surrendering ERC721 Token
+      tokenRegistryInstanceWithShippingLineWallet = await Erc721.new("foo", "bar", { from: shippingLine });
+      tokenRegistryAddress = tokenRegistryInstanceWithShippingLineWallet.address;
+      escrowInstance = await TitleEscrow.new(tokenRegistryAddress, beneficiary1, beneficiary1, ZERO_ADDRESS, {
+        from: beneficiary1,
+      });
+      escrowInstanceAddress = escrowInstance.address;
+      escrowInstance1 = await TitleEscrow.new(tokenRegistryAddress, beneficiary1, beneficiary1, ZERO_ADDRESS, {
+        from: beneficiary1,
+      });
+      escrowInstanceAddress1 = escrowInstance1.address;
+      await tokenRegistryInstanceWithShippingLineWallet.safeMint(escrowInstanceAddress, merkleRoot);
+      await escrowInstance.transferTo(tokenRegistryAddress, {
+        from: beneficiary1,
+      });
+    });
+
+    it("should be able to destroy token", async () => {
+      await tokenRegistryInstanceWithShippingLineWallet.destroyToken(merkleRoot);
+      const currentOwner = tokenRegistryInstanceWithShippingLineWallet.ownerOf(merkleRoot);
+      await expect(currentOwner).to.be.rejectedWith(
+        /VM Exception while processing transaction: revert ERC721: owner query for nonexistent token/
+      );
+    });
+
+    it("non-minter should not be able to destroy token", async () => {
+      const attemptDestroyToken = tokenRegistryInstanceWithShippingLineWallet.destroyToken(merkleRoot, {
+        from: nonMinter,
+      });
+      await expect(attemptDestroyToken).to.be.rejectedWith(
+        /VM Exception while processing transaction: revert MinterRole: caller does not have the Minter role/
+      );
+    });
+
+    it("token cannot be destroyed if not owned by registry", async () => {
+      await tokenRegistryInstanceWithShippingLineWallet.mint(escrowInstanceAddress, merkleRoot1);
+      const attemptDestroyToken = tokenRegistryInstanceWithShippingLineWallet.destroyToken(merkleRoot1);
+      await expect(attemptDestroyToken).to.be.rejectedWith(
+        /VM Exception while processing transaction: revert Cannot destroy token: Token not owned by token registry/
+      );
+    });
+
+    it("should be able to send token owned by registry", async () => {
+      await tokenRegistryInstanceWithShippingLineWallet.sendToken(escrowInstanceAddress1, merkleRoot);
+      const currentOwner = await tokenRegistryInstanceWithShippingLineWallet.ownerOf(merkleRoot);
+      expect(currentOwner).to.deep.equal(escrowInstanceAddress1);
+    });
```

Comment:
Are you really testing a send from the registry?

Reply:
hahah erm, how would you suggest the test to go?

Reply:
sorry misread the code somehow